mu/virtual_environment.py

```diff
@@ -822,6 +825,28 @@ def install_jupyter_kernel(self):
                 "Unable to install kernel\n%s" % compact(output)
             )
 
+    def install_from_unpacked_wheels(self, dirpath):
+        #
+        # This command should install the baseline packages, picking up the
+        # precompiled wheels from the wheels path
+        #
+        # For dev purposes (where we might not have the wheels) warn where
+        # the wheels are not already present and download them
+        #
+        wheel_filepaths = glob.glob(os.path.join(dirpath, "*.whl"))
+        if not wheel_filepaths:
+            raise VirtualEnvironmentCreateError(
+                "No wheels in %s; try `python -mmu.wheels`" % wheels_dirpath
+            )

```

Comment:
Will this ever be raised if a user install Mu from PyPI?

Reply:
tl;dr it shouldn't be. The flow should have failed somewhere earlier, eg, when creating the zip or running `pip download`. By the time we get here, if we have a valid zip which has unpacked, we should have wheels.

That said... nothing's certain, so I'll just change this to a `logger.warning` without the recommendation to run `python -m mu.wheels`